teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -187,6 +189,20 @@ interface TestRunsLocator {
     fun forProject(projectId: ProjectId): TestRunsLocator
     fun withStatus(testStatus: TestStatus): TestRunsLocator
     fun all(): Sequence<TestRun>
+    fun all(fields: EnumSet<TestRunFields>): Sequence<TestRun>
```

Comment:
Without KDoc it isn't evident that this function does. Also this interface uses builder style to avoid adding overloads. So it'll be more consistent and convenient to add a separate function (`limitFields`?) instead. And in this case the function name may be self-documenting so we won't need to add KDoc.